paddle/fluid/framework/channel_impl.h

```diff
@@ -17,6 +17,7 @@ limitations under the License. */
 #include <atomic>
 #include <condition_variable>  // NOLINT
 #include <deque>
+#include <mutex>  // NOLINT
```

Comment:
This file is missing include <mutex> when compile. Just a fix. Not related to this PR.